drakrun/drakrun/sample_startup.py

```diff
@@ -0,0 +1,109 @@
+import logging
+
+import pefile
+import subprocess
+from oletools.olevba import VBA_Parser
+
+from drakrun.vba_graph import get_outer_nodes_from_vba_file
+
+
+log = logging.getLogger("drakrun")
+
+
+class SampleStartupRouter:
```

Comment:
Is there any particular reason to add this class?
IMO we could just put the methods in the module as plain functions.

Reply:
There is no, it was just cleaner for me

Reply:
I mean, `sample_startup` already works as a namespace for those functions. :thinking: 
We can import it with:
```
import drakrun.sample_startup as sample_startup
```
and use with `sample_startup.get_sample_startup_command`

Reply:
fine with me, I will change this